cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -1137,6 +1137,12 @@ func (r *TrafficOpsReq) StartServices(syncdsUpdate *UpdateStatus) error {
 	return nil
 }
 
+func (r *TrafficOpsReq) ShowUpdateStatus(flagType []string, start time.Time, curSetting, newSetting bool) {
```

Comment:
Nitpick, but it isn't clear from the name what "show" is doing, Usually "show" means displaying something to the user in a graphical UI. 
IMO `LogUpdateStatuses` would be clearer